livingdoc-engine/src/test/kotlin/org/livingdoc/engine/execution/examples/scenarios/ScenarioExecutorTest.kt

```diff
@@ -128,7 +128,7 @@ internal class ScenarioExecutorTest {
 
         @Nested inner class `during a setup command (@Before)` {
 
-            @Test fun `the result is Exception`() {
+            @Test fun `the result is "Exception"`() {
```

Comment:
`"` does not compile on windows

Reply:
correction: Does compile on windows with jdk 1.8.161